google-cloud-speech/test/google/cloud/speech/v1/speech_client_test.rb

```diff
@@ -25,7 +25,7 @@
 class CustomTestError < StandardError; end
 
 # Mock for the GRPC::ClientStub class.
-class MockGrpcClientStub
+class MockGrpcClientV1Stub
```

Comment:
@jbolinger Check out these changes I had to make to get the tests passing. The test class created for the V1p1beta test was stepping on the V1 class, leading to test failures. I named all these things differently and that helped get the tests passing.

Reply:
i was seeing failures expecting a V1p1beta1 object but getting a V1 object.

Reply:
Strange, I can't seem to reproduce it. I changed those back let's see what the CI does and I can revert it if needed.